codingStandards/AdditionalGuidelines-Java.md

```diff
@@ -0,0 +1,44 @@
+# Additional Guidelines - Java
+
+## Using assertions
```

Comment:
Section headings should use Title Case like in the other coding standards (linked here for convenience):
- [Java](https://github.com/oss-generic/process/blob/master/codingStandards/CodingStandard-Java.md)
- [CSS](https://github.com/oss-generic/process/blob/master/codingStandards/CodingStandard-Css.md)
- [HTML](https://github.com/oss-generic/process/blob/master/codingStandards/CodingStandard-Html.md)